tutorials/gas_compact_multiphysics/unit_cell.py

```diff
@@ -64,7 +64,7 @@ def coolant_density(t):
     value here in case you want to run these files in standalone mode (i.e. with
     the "openmc" executable).
 
-  Computes the helium density from temperature assuming a fixed operating pressure.
+  Computes the helium density (g/cm3) from temperature assuming a fixed operating pressure.
```

Comment:
i think it'd just be easier to change this comment to `kg/m3` instead of changing all of the rest of the files

Reply:
Yeah that is fair, it is definitely more simple. I'll do this, but I would like to add a warning in tutorial just to make it extra clear for those that live in a g/cm3 world most times.